crates/matrix-sdk-crypto/src/gossiping/mod.rs

```diff
@@ -220,6 +219,7 @@ impl PartialEq for GossipRequest {
 }
 
 #[derive(Debug)]
+#[non_exhaustive]
```

Comment:
Hmmm that's a big change. Why is it needed?

Reply:
Hmm, indeed that's not needed.

Reply:
Removed: https://github.com/matrix-org/matrix-rust-sdk/pull/1365/commits/4152620fe32e182ea9acef3daefda8dcefa7028f